junit-platform-commons/src/main/java/org/junit/platform/commons/util/ResourceUtils.java

```diff
@@ -8,19 +8,21 @@
  * https://www.eclipse.org/legal/epl-v20.html
  */
 
-package org.junit.platform.engine.support.descriptor;
+package org.junit.platform.commons.util;
+
+import static org.apiguardian.api.API.Status.INTERNAL;
 
 import java.net.URI;
 
-import org.junit.platform.commons.util.Preconditions;
-import org.junit.platform.commons.util.StringUtils;
+import org.apiguardian.api.API;
 
 /**
  * Collection of static utility methods for working with resources.
  *
  * @since 1.3
  */
-final class ResourceUtils {
+@API(status = INTERNAL, since = "1.3")
```

Comment:
@marcphilipp Should I leave this as "since 1.3" or change it to the current version?

Reply:
Since it's now public, we should do:
```suggestion
@API(status = INTERNAL, since = "1.12")
```